System/Process.hs

```diff
@@ -870,6 +869,22 @@ It will therefore behave more portably between operating systems than 'system'.
 The return codes and possible failures are the same as for 'system'.
 -}
 rawSystem :: String -> [String] -> IO ExitCode
-rawSystem cmd args = do
-  (_,_,_,p) <- createProcess_ "rawSystem" (proc cmd args) { delegate_ctlc = True }
+rawSystem cmd args = executeAndWait "rawSystem" (proc cmd args) { delegate_ctlc = True }
+
+-- ---------------------------------------------------------------------------
+-- executeAndWait
+
+-- | Create a new process and wait for it's termination.
```

Comment:
Just to confirm my understanding: with the current API on Linux, if we have:

1. Process A calls `rawSystem "B" []`
2. Process B calls `createProcess $ proc "C" []` and then exits

Then the `rawSystem` call in `A` will return when `B` exits, even if `C` is still running. With this new API on Windows, `A` would block until both `B` and `C` exit. Is this correct?